### ISSUE TITLE
iOS app from PGB: 400: An update check must include a binary version that conforms to the semver standard (e.g. '1.0.0').

### ISSUE THREAD

**Issue by ef33zy** [Microsoft/code-push#347]
Hi,

I am using code-push plugin for my app built using PhoneGap Build.

Firstly, the plugin is completely dependent on using cli, which defeats the PGB purpose... nonetheless, I've managed to get the pushing of updates working on Android.

However, when I'm debugging my iOS, I see the below error:
``
[CodePush]  400: An update check must include a binary version that conforms to the semver standard (e.g. '1.0.0'). The binary version is normally inferred from the App Store/Play Store version configured with your app.
``

In my app's `config.xml` file, I have defined the version in the widget as below:
`<widget version = "2.1.0">`

I also have the below config-file element defined.
```
<config-file platform="ios" parent="CFBundleShortVersionString">
<string>210</string>
</config-file>
```

Is the plugin not able to recognize either of these?
Is there something I am missing?
I'm not quite sure how else to define the `binary version` for my app.

Your help & responses are highly appreciated.

**Comment by benjaminb10** [comment]
@ef33zy You need to change your version number 210 in your Info.plist or your config-file to 2.1.0

**Comment by ef33zy** [comment]
Hi @benjaminb10  apologies for not posting earlier.

Since I am building with PhoneGap Build, I removed the below section from my `config.xml` file, and I didn't see the problem anymore.

```
<config-file platform="ios" parent="CFBundleShortVersionString">
<string>210</string>
</config-file>
```

I have also closed out the [issue #187](https://github.com/Microsoft/cordova-plugin-code-push/issues/187) I raised in the cordova-plugin-code-push repository.
